src/test/java/org/cactoos/iterator/NoNullsTest.java

```diff
@@ -48,27 +49,39 @@
 
     @Test
     public void nextThrowsErrorIfNull() {
-        this.exception.expect(IllegalStateException.class);
-        this.exception.expectMessage(
-            new StringContains(
-                "Item #0 of org.cactoos.iterator.NoNullsTest"
-            )
-        );
-        new NoNulls<>(
-            new Iterator<Integer>() {
-                @Override
-                public boolean hasNext() {
-                    return true;
-                }
+        MatcherAssert.assertThat(
```

Comment:
@atapin please use `Assertion` instead of the static methods from `MatcherAssert`